moped-editor/src/views/projects/projectView/ProjectNotes.js

```diff
@@ -316,6 +332,27 @@ const ProjectNotes = ({
     setDeleteConfirmationId(id);
   };
 
+  const handleECaprisSwitch = () => {
+    updateShouldSyncECapris({
+      variables: {
+        projectId: noteProjectId,
+        shouldSync: !shouldSyncFromECapris,
+      },
+    })
+      .then(() => {
+        handleSnackbar(true, "ECAPRIS sync status updated", "success");
+      })
+      .catch((error) =>
```

Comment:
It would probably be good to add a state setter here to flip the UI toggle back to it's previous position if the update fails.